### PR TITLE
trying error logging from within Ruby

### PR DESCRIPTION
**Pull request by regineheberlein**

### CODE REVIEW

**Comment by regineheberlein** [comment]
This is a new attempt at tracing when things go wrong and how. Write output and error to files; add more verbose output; run rubocop; revert unsuccessful logging in scheduler.

File: reports/aspace2alma/get_MARCxml.rb [hunk 1]
```diff
@@ -4,6 +4,14 @@
 require 'net/sftp'
 require_relative '../../helper_methods.rb'
 
+#log to files
```
Comment (maxkadel) [inline reports/aspace2alma/get_MARCxml.rb:7]: Would it be easier to make a logger object? See https://ruby-doc.org/stdlib-3.1.0/libdoc/logger/rdoc/Logger.html#class-Logger-label-How+to+create+a+logger

**Review by maxkadel — CHANGES_REQUESTED** [review]
It looks like the rubocop fixes to the files in `data_fixes/subject_remediation` that I fixed this morning have been put back to how they were before - do they belong in this PR?

File: data_fixes/subject_remediation/get_agents.rb [hunk 2]
```diff
@@ -13,8 +13,9 @@
            :headers => ["title", "created", "used", "source", "type", "ext_id", "string", "uri"]) do |row|
  agent_endpoints.map do |endpoint|
    ids = @client.get(
-     "/agents/#{endpoint}",
-     { query: { all_ids: true } }
+     "/agents/#{endpoint}", {query: {
```
[the remark this answers is not in the document]
Comment (regineheberlein) [reply data_fixes/subject_remediation/get_agents.rb:16]: Ok, this is so weird. I had pulled the changes, ran rubocop, it complained, I -a'ed it, and this is what it did. I just re-ran it, and it's happy. Is this a rubocop config issue?

File: data_fixes/subject_remediation/get_subjects.rb [hunk 3]
```diff
@@ -8,7 +8,9 @@
 output_file = "subjects_test.csv"
 
 subject_ids = @client.get(
-  "/subjects", { query: { all_ids: true } }
+  "/subjects", {query: {
```
[the remark this answers is not in the document]
Comment (regineheberlein) [reply data_fixes/subject_remediation/get_subjects.rb:11]: Yup, ditto.

File: reports/aspace2alma/get_MARCxml.rb (hunk 1 above)
Comment (regineheberlein) [reply reports/aspace2alma/get_MARCxml.rb:7]: I looked at that and it seemed more involved, but if you think it'd be the better way to go, I can keep at it!

**Pull request closed without merging**